docs/markdown/Contributions/releases/release-process.md

```diff
@@ -194,26 +194,10 @@ Then, run:
 ./pants run src/python/pants_release/release.py -- tag-release
 ```
 
-This will tag the release with your PGP key, and push the tag to origin, which will kick off a [`Release` job](https://github.com/pantsbuild/pants/actions/workflows/release.yaml) to build the wheels and publish them to PyPI.
+This will tag the release with your PGP key, and push the tag to origin, which will kick off a [`Release` job](https://github.com/pantsbuild/pants/actions/workflows/release.yaml) to build and publish releases/artifacts.
 
-Step 4: Release a Pants PEX
----------------------------
-
-After the [`Release` job](https://github.com/pantsbuild/pants/actions/workflows/release.yaml) for your tag has completed, you should additionally build and publish the "universal" PEX to Github.
-
-```bash
-PANTS_PEX_RELEASE=STABLE ./pants run src/python/pants_release/release.py -- build-universal-pex
-```
-
-Then:
-
-- Go to <https://github.com/pantsbuild/pants/tags>, find your release's tag and click `Create release from tag`.
-- If this is not the latest stable release, deselect "Set as the latest release".
```

Comment:
I suspect we can still automate this. Doing semver comparison with the last N releases (IDK like 30?)

Reply:
Actually:

>       --latest                       Mark this release as "Latest" (default: automatic based on date and version)

So I think we're good to go there!